src/utils/authConfig.ts

```diff
@@ -2,9 +2,9 @@
 // visit https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/configuration.md
 export const msalConfig = {
   auth: {
-    clientId: process.env.REACT_APP_AUTH_CLIENT_ID || '',
-    authority: `https://login.microsoftonline.com/${process.env.REACT_APP_AUTH_TENANT_ID}`,
-    redirectUri: process.env.REACT_APP_AUTH_REDIRECT_URI || 'http://localhost:3000',
+    clientId: window._env_.REACT_APP_AUTH_CLIENT_ID || '',
+    authority: `https://login.microsoftonline.com/${window._env_.REACT_APP_AUTH_TENANT_ID}`,
+    redirectUri: window._env_.REACT_APP_AUTH_REDIRECT_URI || 'http://localhost:3000',
```

Comment:
Does not work in development mode.
![image](https://user-images.githubusercontent.com/3524687/96337463-0a5af400-1090-11eb-9dc9-17fdcaf6a6d0.png)